src/maingoblint.ml

```diff
@@ -57,6 +57,11 @@ let create_temp_dir () =
 let remove_temp_dir () =
   if not (get_bool "keepcpp") then ignore (Goblintutil.rm_rf !Goblintutil.tempDirName)
 
+
+let rec print_list = function 
+  [] -> ()
+  | e::l -> print_int e ; print_string " " ; print_list l
+
```

Comment:
Is this still needed for anything or just left over from some debugging?

Reply:
Leftover from debugging, sorry.